extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -645,11 +642,8 @@ export class AmpStoryPage extends AMP.BaseElement {
           }
           this.cssVariablesStyleEl_.textContent =
             `:root {` +
-            `--story-page-vh: ${px(state.vh)};` +
-            `--story-page-vw: ${px(state.vw)};` +
-            `--story-page-vmin: ${px(state.vmin)};` +
-            `--story-page-vmax: ${px(state.vmax)};` +
-            `--i-amphtml-story-page-50vw: ${px(state.fiftyVw)};` +
+            `--story-page-vh: ${px(state.vh)} !important;` +
+            `--story-page-vw: ${px(state.vw)} !important;` +
```

Comment:
Why don't we go 100% CSS? I'd like it if we could remove this code entirely. It's just duplicating the CSS and at worst can cause CLS for no good reason